db/seeds.rb

```diff
@@ -1 +1,75 @@
+class Seed
+  def run
+    create_borrowers(10)
+    create_lenders(10)
+    create_loan_requests_for_each_borrower(3)
+    create_categories(7)
+  end
 
+  def lenders
+    User.where(role: 0)
+  end
+
+  def borrowers
+    User.where(role: 1)
+  end
+
+  def create_lenders(quantity)
+    quantity.times do
+      name = Faker::Name.name
+      email = Faker::Internet.email
+      user = User.create(name: name, password: 'password', email: email, role: 0)
+      puts "created user #{user.name}"
+    end
+  end
+
+  def create_borrowers(quantity)
+    quantity.times do
+      name = Faker::Name.name
+      email = Faker::Internet.email
+      user = User.create(name: name, password: 'password', email: email, role: 1)
+      puts "created user #{user.name}"
+    end
+  end
+
+  def create_categories(quantity)
+    quantity.times do
+      title = Faker::Lorem.words(2).join(" ")
+      description = Faker::Lorem.sentence
+      category = Category.create(title: title, description: description)
+      puts "created category #{category.title}"
+    end
+    put_requests_in_categories
+  end
+
+  def put_requests_in_categories
+    LoanRequest.all.shuffle.each do |request|
+      Category.all.shuffle.first.loan_requests << request
+      puts "linked request and category"
+    end
+  end
+
+  def create_loan_requests_for_each_borrower(quantity)
```

Comment:
Assignment Branch Condition size for create_loan_requests_for_each_borrower is too high. [25.96/15]